core/src/test/java/org/elasticsearch/index/query/RandomQueryBuilder.java

```diff
@@ -47,6 +48,8 @@ public static QueryBuilder createQuery(Random r) {
             case 3:
                 return createMultiTermQuery(r);
             case 4:
+                return new GeoDistanceRangeQueryTests().createTestQueryBuilder();
```

Comment:
This is not related to your PR but to me it looks weird that we need to create an instance of a test class to get a query builder. I think we should change that to a cleaner API (but not in this PR).